templates/gatsby-drupal/files/README.md

```diff
@@ -0,0 +1,86 @@
+2. Once you have completed the Drupal install, you will need to enable a few plugins in order to fully configure it to be used as a backend for the Gatsby site. From the admin page, install the following from **Extend**:
+
+- Gatsby JSON:API Extras
+- Gatsby JSON:API Instant Preview and Incremental Builds
+- JSON API
+- JSON API Extras
+- Serialization
+- Pathauto
+
+3. Set up aliases for your articles. From **Configuration**, go to the subsection *Search and Metadata* and click on *URL aliases*. With Pathauto enabled, you should be able to click on a header called *Pattern*. Click *Add Pathauto pattern*, and set it with the following fields:
+    - Pattern type: Content
+    - Path pattern: `/articles/[node:title]`
+    - Content type: Article
+    - Label: `pathautho-article`
+    - Enabled
+
+  Save the pattern. Now *Articles*, which would typically be available at `/node/<NODE NUMBER>` will have an alias at `/articles/<slug-of-article-title>`, which will be replicated on the final Gatsby site.
+
+4. Add *Articles* by clicking the **Content** tab. Make sure to include all of the required fields in *Articles* (title, summary, body, image, and alt text for the image).
+
+> **Note:** If you plan on enabling Gatsby Live Preview on your development environments, make sure to add at least two Articles now (larger description in Live Preview section below).
+
+5. With the above steps completed, redeploy the environment with `platform redeploy -p <PROJECT ID> -e master`. The environment will redploy, and the Gatsby site will pull data from the backend Drupal API.
+
+### Enabling Gatsby Live Preview (manual configuration)
+
+> **Note:** Live Preview is not enabled by default on *Master* environments, but it can be set up manually on your development environments.
+> **Note:** After Live Preview is enabled, you will be able to edit content within Drupal, which will cause that content to update on Gatsby automatically for changes to its title, summary and body. Make sure to have at least two articles in Drupal before attempting to update an article's image. Gatsby temporarily is not able to locate the article during the update, and the environment will have to be redeployed. As long as there is more than one article present, this problem will not be encountered.  
+
+0. Branch off of master a new development environment.
+1. Within the Drupal admin on that development envirnonment, click on *Gatsby Settings* in **Configuration**/**Web Services** section.
+2. Update the following fields and save the new configuration:
+  - Gatsby Preview Server URL: the root url for your environment (`https://<dev-env-name>-<hash>-<projectId>.<region>.platformsh.site/`)
```

Comment:
Gross. :-)  Is there no semi-automated way to do that?  Say... a drush command that can set the config value based on the env var?

Reply:
This is the one that I've been playing with, and would still like to fix. But for now, it doesn't seem that the module will accept the override for it. I figure that we can  merge this far - which is as far as I've ever demo'd anyway, and then I can keep trying  to get it automatic on each branch.

Reply:
Hrm.  OK for now, I guess, but we should at least be able to just run `drush config set blah blah` or something in the deploy hook, no?  Or is the module just that bad?

Reply:
That's what I was working with, but the module wasn't picking it up yeah.

Reply:
Wow, the module is just that bad.  Good to know. :sigh: